launchlib/launcher.go

```diff
@@ -109,24 +99,11 @@ func compileCmdFromConfig(
 			}
 			args = append(args, executable) // 0th argument is the command itself
 
+			// Filter out any heap-related options that are not supported by the current mode (container mode enabled/disabled), and adds -XX:MaximumHeapSizePercent using heapPercentage if applicable
```

Comment:
This seems to imply that we're _only_ filtering heap-related options, but that's not all we're doing, right? We're also applying a whitelist to the jvm opts